src/main/java/wolox/training/repositories/UserRepository.java

```diff
@@ -0,0 +1,12 @@
+package wolox.training.repositories;
+
+import javax.validation.constraints.NotNull;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.CrudRepository;
+import wolox.training.models.User;
+
+public interface UserRepository extends CrudRepository<User, Long> {
+
+    @Query(value = "SELECT u FROM User u WHERE u.username = ?1")
+    User findByUsername(@NotNull String username);
```

Comment:
This method should return a wrapped object rather than a User, in order to avoid to return null values: `Optional<User>`